registry/proxy/proxyblobstore.go

```diff
@@ -67,6 +68,26 @@ func (pbs *proxyBlobStore) copyContent(ctx context.Context, dgst digest.Digest,
 	return desc, nil
 }
 
+// This method is intended to avoid actual datacopy of Head request.
```

Comment:
Please do this like most godoc comments `// doServeBlobResponse ....`.

I know this file doesn't really follow that well, but most of this code base has godoc style comments, whether or not the symbol is exported.